docs/installation.md

```diff
@@ -22,9 +22,9 @@ Similarly, you can run the latest 3.* version with:
 ```shell
 docker run -it -v $(pwd):/config ghcr.io/gitlabform/gitlabform:3 gitlabform
 ```
-* [pipx](https://github.com/pypa/pipx):
+* [uv](https://github.com/astral-sh/uv):
 ```shell
-pipx run --spec 'gitlabform>=3,<4' gitlabform
+uvx --from 'gitlabform>=3,<4' gitlabform
```

Comment:
This isn't really about `uv`. Looks like we already had constraint about gitlabform version being lower than 4. Not really familiar with either commands or what it's for. Should this be changed, now that we're already on version 4? But... what's the point of having a constraint? If we don't specify it, will it automatically install the latest one?